framework/include/base/MooseApp.h

```diff
@@ -521,6 +521,29 @@ class MooseApp : public ConsoleStreamInterface,
    */
   bool halfTransient() const { return _half_transient; }
 
+  /**
+   *  Set the passed variable's value to that of the named command line parameter (if valid)
+   *
+   * @param variable_to_set The variable whose value is to be modified
+   * @param cli_param_name The name of the command line parameter used to set variable_to_set
+   *
+   * @return variable_value_changed Whether the passed variable was changed (whether cli_param_name
+   * is valid)
+   */
+  template <typename T>
+  bool setVariableToCommandLineParam(T & variable_to_set, const std::string & cli_param_name) const
```

Comment:
I think this is a very specific use case that doesn't necessitate a new method for it. Can you just put this logic into the method in `Console` instead? You can access `MooseApp::isParamValid()` from the `Console`

Reply:
Did you mean `Output` (not `Console`)?

Reply:
correct

Reply:
So this no longer exists, right?